usr.sbin/bsdinstall/scripts/auto

```diff
@@ -405,10 +410,6 @@ finalconfig() {
 		bsdinstall services
 		finalconfig
 		;;
-	"System Hardening")
```

Comment:
You haven't dropped the entry from the dialog call but have dropped it here. I'd leave this code and just remove it from the menu. Or leave it there for people who still want to use it, just not part of the default path through the installer. That's probably best?

Reply:
Yes, sounds good. Well, I’d rather remove it entirely from bsdinstall, but that should go upstream first. I’ll simply trim default presentation of the menu during install.